app/views/deliveries/index.html.erb

```diff
@@ -0,0 +1,103 @@
+<div>
+  <div>
+    <h1>
+      Delivery Tracker
+    </h1>
+
+    <p>Know if something gets lost in the mail.</p>
+  </div>
+</div>
+
+<hr>
+
+<div>
+  <div>
+    <h2>
+      Expecting a package?
+    </h2>
+
+    <form action="/insert_delivery" method="post">
+      <div>
+        <label for="description_box">
+          Description
+        </label>
+
+        <input type="text" id="description_box" name="query_description">
+      </div>
+
+      <div>
+        <label for="supposed_to_arrive_on_box">
+          Supposed to arrive on
+        </label>
+
+        <input type="date" id="supposed_to_arrive_on_box" name="query_supposed_to_arrive_on">
+      </div>
+
+      <div>
+        <label for="details_box">
+          Details
+        </label>
+
+        <textarea id="details_box" name="query_details" rows="3"></textarea>
+      </div>
+
+      <div>
+        <input type="hidden" id="user_id_box" name="query_user_id" value="<%= current_user.id %>">
```

Comment:
this is one way to assign the user id, but could be a security risk. Assigning it in the controller would be a better way of doing it :)